src/Slack/SlackClient.ts

```diff
@@ -68,15 +70,25 @@ export class SlackClient implements ISlackClient, IErrorReporter {
     if (candidate.linkedin) {
       links.push(`<${candidate.linkedin}|LinkedIn>`);
     }
-    links.push(`<${candidate.details}|${candidate.source ? candidate.source : 'StartupJobs'}>`);
+    if (candidate.linkedin) {
```

Comment:
Why does this check for another `candidate.linkedin` when there is already one if condition above?